packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -397,7 +397,7 @@ export class MetriportMedicalApi {
    * Returns the document query status for the specified patient.
    *
    * @param patientId Patient ID for which to retrieve document query status.
-   * @return The document query progress & status indicating whether its being executed or not.
+   * @return The document query ID, progress & status indicating whether its being executed or not.
```

Comment:
don't we have to add the requestId to the schema? https://github.com/metriport/metriport/blob/cc4a1ef03e1f260f484533e51096bde23807e116/packages/api-sdk/src/medical/models/document.ts#L39-L42

assuming this wasn't tested with the SDK?

Reply:
tested now